tools/capacity-planner-sheets-extension/sheets.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * 
+ * Convenience functions for creating, writing to, and formatting sheets.
+ */
+
+BACKGROUND_COLORS = ["#ead1dc", "#cfe2f3", "#d0e0e3", "#fff2cc", "#f4cccc"];
```

Comment:
This variable is not used, consider removing it if that's the case. If you're keeping it, consider setting it as `const`